Brokerages/InteractiveBrokers/InteractiveBrokersBrokerage.cs

```diff
@@ -1647,6 +1649,13 @@ public IEnumerable<BaseData> GetNextTicks()
 
             foreach (var tick in ticks)
             {
+                // if we have a cached data time zone for this symbol, perform the conversion 
+                DateTimeZone dataTimeZone;
+                if (_dataTimeZones.TryGetValue(tick.Symbol.Value, out dataTimeZone))
+                {
+                    tick.Time = tick.Time.ConvertTo(TimeZones.NewYork, dataTimeZone);
```

Comment:
Data time zone is the raw tone zone from the data source and exchange time zone is the time zone of data sent into the algorithm. So I was expecting us to convert from data time zone to the exchange home zone instead of new York tz